app/views/lti/choose_course.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="lti-flash">
+  <%= render partial: 'shared/flash_message' %>
+</div>
+<div id="course_selector">
+  <div id="lti_course_info">
+    <h3><%=t('lti.select_course', course_name: session[:canvas_course_name]) %></h3>
+  </div>
+  <%= form_with(local:true) do |f| %>
+  <div class="table">
+    <table>
+      <thead>
+      <tr>
+        <th><%= Course.model_name.human %></th>
+      </tr>
+      </thead>
+      <tbody>
+      <% @real_user.visible_courses.each do |course| %>
```

Comment:
this will show courses where the user is a student/TA, not just instructor (please fix and add tests)